com.unity.shadergraph/Editor/Data/Nodes/NodeClassCache.cs

```diff
@@ -60,10 +60,10 @@ public static IEnumerable<ContextFilterableAttribute> GetFilterableAttributesOnN
         private static void ReCacheKnownNodeTypes()
         {
             Profiler.BeginSample("NodeClassCache: Re-caching all known node types");
-            m_KnownTypeLookupTable = new Dictionary<Type, List<ContextFilterableAttribute>>();
+            m_KnownTypeLookupTable.Clear();
```

Comment:
@pbbastian wait, if this gets auto-gc'd then why is it better to do this, when on assembly reload we will have to call new anyways?

Reply:
we cant just clear it at this point since its null at this point

Reply:
oh right, that's a good point!